src/website/views/custom-elements/spinner.pug

```diff
@@ -0,0 +1,82 @@
+---
+title: Spinner
+scripts:
+  - js/ce/ux-chi-ce
+---
+
+p.-text Spinners are animated circles that let users know when information is saving or loading.
+
+h3 Examples
+p.-text To render a spinner, use <code>chi-spinner</code> tag.
+
+h3 Backdrop
+p.-text Wrap spinners in <code>a-backdrop</code> to overlay content similar to a modal.
+h4 Base
+.example.-mb--3
+  .a-backdrop.-p--6(style="position:relative;height:25rem;")
+    chi-spinner(color='inverse', size='lg')
+  :code(lang="html")
+    <div class="a-backdrop">
```

Comment:
Use backdrops as an attribute instead of wrapping the element with it. Please, make this update in the rest of the document.